src/SenseNet.Security/PermissionEvaluator.cs

```diff
@@ -226,6 +227,9 @@ internal PermissionValue GetSubtreePermission(int userId, int entityId, int owne
 
         internal List<AceInfo> GetEffectiveEntries(int entityId, IEnumerable<int> relatedIdentities = null, EntryType? entryType = null)
         {
+            if (_securityContext.CurrentUser.Id == Identities.SystemUserId)
+                return AceInfo.GetElevatedAces(entryType);

```

Comment:
Please check if this correct, because this changes the previous behavior: it returns a single admin aceinfo instead of a whole list for real users and groups. This should be a query method.